### DISCUSSION TITLE
Settings to define the max memory allocated

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#306] [Feedback & Feature Proposal]
Should solve this bug (only on Windows): https://github.com/meilisearch/MeiliSearch/issues/1899

We could add a setting to set the maximum memory allocated.

FYI, this feature is already implemented on Milli's side but not exposed on MeiliSearch's side yet.

**Comment by gmourier** [comment]
For v0.27.0, we plan to add two more options.

- `--max-memory` (`MEILI_MAX_MEMORY` env var) (Directly related to this product discussion)
- `--indexing-jobs` (`MEILI_INDEXING_JOBS` env var)

[Specification link](https://github.com/meilisearch/specifications/pull/119) to get more details.

**Reply by danielliusc** [reply]
May I know whether such configurations are exposed for meilisearch server? Such configurations are not found when I try to set. I used the meilisearch server v0.26.1 for windows. Or such configuration should be set by some other way?

**Reply by gmourier** [reply]
Hi @danielliusc 👋

These configuration options are not yet released. They will be available in the next version (v0.27) which is expected on May 9th 2022.

**Reply by danielliusc** [reply]
it is great !!

**Comment by gmourier** [comment]
Following the [v0.27 release](https://github.com/meilisearch/meilisearch/releases/tag/v0.27.0), this discussion is now locked.

We encourage you to create a new thread if needed.

Thank you ✨
